test/draw_test.py

```diff
@@ -374,21 +374,144 @@ def todo_test_circle(self):
 
         self.fail()
 
-    def todo_test_polygon(self):
 
-        # __doc__ (as of 2008-08-02) for pygame.draw.polygon:
+RED = (255, 0, 0)
```

Comment:
`pygame.Color('red')` is a good alternative to these color constants.

Reply:
done